Game/PlayerPanel.gd

```diff
@@ -27,6 +26,12 @@ func _ready():
 	set_process_unhandled_input(true)
 	
 	camera3d.current = true
+	
+func _physics_process(_delta):
+	if visible:
+		stage3d.translation.x = 0
+	else:
+		stage3d.translation.x = 1000
```

Comment:
Well it's just an angry try to solve one problem:
Because all UI's running all together, GlobalUIPopup shows info about player's part even it's covered by other control)